lib/liftoff/scheme_builder.rb

```diff
@@ -0,0 +1,50 @@
+module Liftoff
+  class SchemeBuilder
+
+    def initialize(xcode_project, config)
+      @xcode_project = xcode_project
+      @config = config
+    end
+
+    def create_schemes
+      @xcode_project.generate_scheme # generates the default scheme
+    
+      # generates custom schemes
+      config_schemes.each do |scheme_config|
+        name = name_from_scheme_config(scheme_config)
+        generate_scheme(name, @xcode_project, scheme_config["actions"])
+      end
+    end
+
+    private
+
+    def config_schemes
+      @config.schemes || []
+    end
+
+    def name_from_scheme_config(scheme_config)
+      string_renderer.render(scheme_config["name"])
+    end
+
+    def generate_scheme(name, project, actions)
+      actions ||= []
+      project.generate_scheme(name) do |scheme|
+        actions.each do |action, action_config|
+          add_action_to_scheme(action, action_config, scheme)
+        end
+      end
+    end
+
+    def add_action_to_scheme(action, action_config, scheme)
+      action_elem = scheme.doc.root.elements["#{action.capitalize}Action"]
```

Comment:
What do you think about creating a method to get the build configuration from a scheme and use it here?